docker/Dockerfile.http

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:20.04
+
+# Get arguments from the build command line
+ARG version
+ENV VERSION=$version
+
+# Build up the OS
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update && \
+    apt-get install -y curl \
+                       libtcnative-1 \
+                       maven \
```

Comment:
I don't see any `RUN mvn` statements later in the file; is `maven` needed at run time?

Reply:
You're right. Maven is not needed. Then we definitely should replace custom Ubuntu image with standard JDK image, for example, `FROM openjdk:11-slim`

Reply:
Woot woot! 🎊

Reply:
openjdk:11-slim works well - builds/loads blazingly fast since it's so compact. Thanks for the suggestions!